src/main/java/com/szeloch/jan/parkingsystemmanagement/rest/model/CreditCard.java

```diff
@@ -0,0 +1,31 @@
+package com.szeloch.jan.parkingsystemmanagement.rest.model;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import java.util.Currency;
+
+@Data
+@Entity
+@NoArgsConstructor
+public class CreditCard {
+
+    @Id
+    @GeneratedValue
+    private Long id;
+
+    private Currency currency;
+
+    private Long creditCardNumber;
```

Comment:
Czemu long?

Reply:
Hmm, pewnie lepiej dać String. Numer karty to 16 cyfr, Integer ich nie zapisze.

Reply:
Generalnie jeśli nie mnożysz, nie dodajesz albo nie liczysz całek, to lepiej jest robić string. To samo tyczy się wszystkich PESEL-i, VIN-ów itp. To że dziś karty reprezentowane są przez cyferki to tylko czyjaś decyzja, kto wie, może za rok na numerach kart pojawią się literki?

Reply:
Rozumiem.